tests/Aspire.Hosting.Azure.Tests/ExistingAzureResourceTests.cs

```diff
@@ -1005,9 +751,7 @@ public async Task SupportsExistingAzureSignalRWithResourceGroup()
               "connectionString": "Endpoint=https://{signalR.outputs.hostName};AuthType=azure",
```

Comment:
Can we add a snapshot test for the roles file in this case? I don't think we need to set a scope on the roles bicep (since we're declaring the role assignments in the same resource group as the app) but we might want to verify that in a test.

Reply:
Opened [Add test to ensure the scope is set correctly on an existing resource's role assignments bicep. (dotnet/aspire#8353)](https://github.com/dotnet/aspire/pull/8353) for this.

> I don't think we need to set a scope on the roles bicep (since we're declaring the role assignments in the same resource group as the app) but we might want to verify that in a test.

We do need to set a scope on the roles bicep, since we need the `existing` resource in order to add role assignments to it.

So just like before, where the role assignments were in the same bicep module as the "existing" resource, with the scope set, we need to set the scope on the new resource as well.